packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -207,25 +211,18 @@ export interface GridOptions {
    * @default debug
    */
   logLevel?: string | false;
-
-  /**
-   * Callback fired when the edit cell value changed.
-   * @param handler
-   */
-  onEditCellChange?: (params: GridEditCellPropsParams, event?: React.SyntheticEvent) => void;
   /**
-   * Callback fired when the cell changes are committed.
-   * @param handler
+   * Callback fired when the edit cell value changes.
+   * @param {GridEditCellPropsParams} params With all properties from [[GridEditCellPropsParams]].
+   * @param {React.SyntheticEvent} event The event that caused this prop to be called.
    */
-  onEditCellChangeCommitted?: (
-    params: GridEditCellPropsParams,
-    event?: React.SyntheticEvent,
-  ) => void;
+  onEditCellPropsChange?: (params: GridEditCellPropsParams, event?: React.SyntheticEvent) => void;
   /**
-   * Callback fired when the EditRowModel changed.
-   * @param handler
+   * Callback fired when the cell changes are going to be committed.
+   * @param {GridEditCellPropsParams} params With all properties from [[GridEditCellPropsParams]].
+   * @param {React.SyntheticEvent} event The event that caused this prop to be called.
    */
-  onEditRowModelChange?: (params: GridEditRowModelParams) => void;
+  onCellEditCommit?: (params: GridEditCellPropsParams, event?: React.SyntheticEvent) => void;
```

Comment:
According to our event naming convention, we must use the present tense. Here the prop was renamed to follow the event it listens to.  We don't need to rename other props because this was the only one that was wrong.

Reply:
I meant that we will have https://next.material-ui.com/api/slider/ `onChangeCommitted` to rename.